common/flatpak-dir.c

```diff
@@ -16700,12 +16708,14 @@ flatpak_dir_list_unused_refs (FlatpakDir         *self,
     }
 
   used_refs = g_hash_table_new_full ((GHashFunc)flatpak_decomposed_hash, (GEqualFunc)flatpak_decomposed_equal, (GDestroyNotify)flatpak_decomposed_unref, NULL);
+  autoprune_refs = g_hash_table_new_full ((GHashFunc)flatpak_decomposed_hash, (GEqualFunc)flatpak_decomposed_equal, (GDestroyNotify)flatpak_decomposed_unref, NULL);
 
-  g_info ("Checking installation ‘%s’ %s",
+  g_info ("Checking installation ‘%s’ %s %s",
           flatpak_dir_get_name_cached (self),
-          filter_by_eol ? "for EOL unused refs" : "for unused refs");
+          filter_by_eol ? "for EOL unused refs" : "for unused refs",
+          filter_by_autoprune ? "and autoprunes" : "");
```

Comment:
```suggestion
          filter_by_autoprune ? " and autoprunes" : "");
```

to avoid a trailing space when not filtering by autoprune